examples/pennylane_skip_plugins_hybrid.py

```diff
@@ -0,0 +1,391 @@
+r"""
```

Comment:
For plugins with `skip`, what is the workflow if we actually change those files and want to update the notebook/pages correctly?

Reply:
Also, can we put `skip` at the end?

Reply:
> For plugins with skip, what is the workflow if we actually change those files and want to update the notebook/pages correctly?

These scripts will always be converted and built if Sphinx Gallery detects a change. The only thing it won't do is execute them to generate output (so output cells will have to be manually updated, like we do now with the ReST/Jupyter notebook tutorials).

> Also, can we put skip at the end?

I could look into removing skip altogether. Instead, I could configure it to only run tutorial scripts _containing_ a word of phrase (such as 'run').

Reply:
I've changed it so that `pennylane_run_*` are executed, and `pennylane_` (without the run) are not.